scripts/install-windows-prereqs.ps1

```diff
@@ -588,28 +589,82 @@ function Install-Docker {
     Write-Output '[WARNING] Docker is no longer a prerequisite and installation thereof is deprecated.'
 }
 
-try {
+function Validate-SkipTasks {
+    Param(
+        [Parameter(Mandatory=$true)][array]$SkipTaskLower
+    )
+    $ValidSet = @("Nuget3", "Nuget6", "Python", "VisualStudio", "LLVM", "Git", "Shellcheck", "NSIS", "Docker", "OpenSSL", "VCRuntime")
+    foreach($item in $SkipTaskLower) {
+        if($ValidSet -notcontains $item) {
+            Write-Output "$item is not a valid package to skip."
+            return
+        }
+        Write-Output "Skipping $PackageToInstall"
+        return
+    }
+}
 
+try {
     if ($VerificationOnly) {
         Write-Output 'Verification only - Complete'
         Exit 0
     }
-
+    $lowercaseSkipTask = $SkipTask.ForEach({$_.ToLower()})
```

Comment:
Suggestion: Remove this line and refactor to use case-insensitive matching instead. https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-5.1

E.g. `-notcontains` -> `-inotcontains`